examples/acc_regional_simulation.jl

```diff
@@ -0,0 +1,219 @@
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.Grids: φnode
+using ClimaOcean
+using ClimaOcean.ECCO
+
+using Printf
+using CairoMakie
+using CFTime
+using Dates
+
+arch = GPU() 
+
+Nx = 1440
+Ny = 600
+Nz = 40  
+
+z_faces = exponential_z_faces(Nz=Nz+1, depth=6000)
+
+grid = LatitudeLongitudeGrid(arch;
+                             size = (Nx, Ny, Nz),
+                             halo = (7, 7, 7),
+                             z = z_faces, 
+                             latitude  = (-80, -20),
+                             longitude = (0, 360))
+
+bottom_height = regrid_bathymetry(grid; 
+                                  minimum_depth = 10,
+                                  interpolation_passes = 5,
+                                  major_basins = 1)
+ 
+grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bottom_height), active_cells_map=true) 
+
+dates = DateTimeProlepticGregorian(1993, 1, 1) : Month(1) : DateTimeProlepticGregorian(1993, 5, 1)
```

Comment:
We are using only 5 months for restoring here.
Therefore, the restoring and the forcing go out of sync passed 5 months and we create strong density gradients in the northern sponge region. Most likely we should include a whole year.
```suggestion
dates = DateTimeProlepticGregorian(1993, 1, 1) : Month(1) : DateTimeProlepticGregorian(1993, 12, 1)
```